docs/dg/projectManagement.md

```diff
@@ -37,10 +41,11 @@ Take note of the following when making the release according to the above guide:
   * Append `rc` to the version number to indicate that the release is a pre-release that is not ready to be used in production.
 * Enter the release title as `RepoSense vxxx` where `xxx` is the version number. Enter the release description by referring to the previous [RepoSense releases](https://github.com/reposense/RepoSense/releases).
 * Before launching the release, generate the `RepoSense.jar` file and attach it to the release.
-  * Change the directory to the project root directory.
-  * In the terminal, run `gradlew --version` to check that the JDK version is 1.8.0.
-  * Run `gradlew shadowJar`, and the Jar file will be generated at `{buildDir}/jar/`.
-  * Check that the Jar file is working. You may need to check that the report can be generated from the Jar file both locally and remotely by following the [Generating Reports Guide](../ug/generatingReports.html).
+  1. Switch to the `release` branch.
+  1. In the terminal, Change the directory to the project root directory.
```

Comment:
```suggestion
  1. In the terminal, change the directory to the project root directory.
```

Lower-case after comma

Reply:
Thanks for noticing this. The adjustment has been made